PrincipleStudios.Tools.Ui/src/components/headings/index.tsx

```diff
@@ -27,15 +27,25 @@ const header =
 		);
 
 export const Headings = {
-	h1: header('h1', 'text-3xl'),
-	h2: header('h2', 'text-2xl'),
-	h3: header('h3', 'text-xl'),
-	h4: header('h4', 'text-lg'),
-	h5: header('h5', 'text-base'),
-	h6: header('h6', 'text-sm'),
+	h1: header('h1', 'text-4xl'),
+	h2: header('h2', 'text-3xl'),
+	h3: header('h3', 'text-2xl'),
+	h4: header('h4', 'text-xl'),
+	h5: header('h5', 'text-lg'),
+	h6: header('h6', 'text-base'),
 	// h7 to be used because our mdx down-steps all header tags intentionally
-	h7: header('h6', 'text-xs'),
+	h7: header('h6', 'text-sm'),
+	h8: header('h6', 'text-xs'),
 
 	byNumber: (n: number) =>
 		Headings[`h${Math.floor(clamp(1, 7, n)) as 1 | 2 | 3 | 4 | 5 | 6 | 7}`],
```

Comment:
You've added an h8. Do you need to change this math at all?

Reply:
Also curious, what's the reason behind h7, and 8 when the html spec is 1 through 6?

Reply:
Yep, I do need to fix the typings there, good catch, @meganbednarz!

This is a little trick I use to help with nested `mdx` files, as the comment says:
> h7 to be used because our mdx down-steps all header tags intentionally
The idea is that we want one h1 per page, and want to be able to embed the content of an MDX either a) as its own page or b) in the middle of another article. In the latter case, the `h1` of the MDX would become an `h2`, the `h2`'s would become an `h3`... and we want to still support further subheadings if necessary. (So far it hasn't come to that!)